config.py

```diff
@@ -13,12 +13,12 @@ class RegConfig:
     lr = 1e-3
     epochs = 1000
     train_samples = 5
-    test_samples = 100
+    test_samples = 10
     num_test_points = 400
     mode = 'regression'
     mixture_prior = False
-    hidden_units = 400
-    noise_tolerance = .1
+    hidden_units = 600              # CHANGED TO 100
```

Comment:
Could we leave the config settings as they were to reflect the values that were used to create the final saved models/results that will be included in the report?

Reply:
done.